Website/admin/ControlPanel/ControlBar.ascx

```diff
@@ -435,6 +435,10 @@
         $('a#ControlBar_ViewInPreview').click(function() {
             <%=PreviewPopup() %>;
         });
+
+        <% if (IsBeaconEnabled) { %>
+        (new Image()).src = "<%= GetBeaconUrl() %>";
```

Comment:
this should be properly encoded (e.g. `(new Image()).src = "<%=Localization.GetSafeJSString(GetBeaconUrl())%>"`